### PR TITLE
CM Tab Update

### PR DESCRIPTION
**Pull request by mmaxsong**
for anh-thu, max, and celeste!

### CODE REVIEW

File: committee.html [hunk 1]
```diff
@@ -983,25 +983,20 @@ <h1>THE COMMITTEE</h1>
...
-            involved with the Cal ALCU on campus, and CUSD Against Racism off campus! On any given
-            day you might find me at Trader Joe’s loading up on snacks, listening to Fleetwood Mac,
-            hiking, cheffing it up in my dorm kitchen, and hanging out with my awesome floor at Unit
-            1. Berkeley is so much fun and I am beyond excited to get to meet and share it with all
-            of you! If you have any questions about anything (classes, majors, RCSA, best food hehe)
-            feel free to reach out! Can’t wait to meet all of you!
+            Celeste is a freshman in Society & Environment, and she's interested in all 
+            things environmental policy and agricultural sustainability! She's from Maryland 
+            (DC area) and will gladly explain the concept of snow and ~weather~ to any 
+            alifornians :) She loves the outdoors, any sort of running around, cooking &
```
Comment (KhanWhale) [inline committee.html]: there's a typo in her bio - says 'alifornians'. Can you fix this?

**Review by KhanWhale — CHANGES_REQUESTED** [review]
See my comments

File: committee.html (hunk 1 above)
Comment (mmaxsong) [reply committee.html]: Yes!

**Review by KhanWhale — APPROVED** [review]
Looks good to me!

**Pull request merged**